src/ConnectionConfig.test.tsx

```diff
@@ -176,7 +177,12 @@ describe('ConnectionConfig', () => {
     fireEvent.click(gceAuthButton);
 
     expect(onOptionsChangeSpy).toHaveBeenCalledWith({
-      jsonData: { authenticationType: GoogleAuthType.GCE },
+      jsonData: {
+        authenticationType: GoogleAuthType.GCE,
+        clientEmail: 'test@grafana.com',
+        defaultProject: 'test-project',
+        tokenUri: 'https://accounts.google.com/o/oauth2/token',
```

Comment:
nit: It'd be nice to have these properties in a object outside the test so  it's not needed to repeat it every time.